lib/api_auth/base.rb

```diff
@@ -22,22 +22,24 @@ class << self
     # access_id: The public unique identifier for the client
     #
     # secret_key: assigned secret key that is known to both parties
-    def sign!(request, access_id, secret_key)
+    def sign!(request, access_id, secret_key, options = {})
+      options = { :override_http_method => nil, :with_http_method => false }.merge(options)
```

Comment:
Oops, yeah I wanted it to be false, not nil. I'll update it. I still want the merge there to make it clear what the available options are very close to the method definition. I'd just use keyword arguments if we weren't still trying to be 1.8.7 compatible.